submissions/okkkko/a_tiny_js_world/index.js

```diff
@@ -0,0 +1,80 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+
+const dog = {	
+	species:"dog",
+	name:"Rex",
+	gender:"male",
+	legs:4,
+	hands:0,
+	saying:"WOOF!",
+	friends: ["Joey"," Lisa"]
+};
+const cat = {
+	species:"cat",
+	name:"Milka",
+	gender:"female",
+	legs:4,
+	hands:0,
+	saying:"meeeow!",
+	friends: ["Selina"]	
+};
+const man = {
+	species:"human",
+	name:"Joey",
+	gender:"male",
+	legs:2,
+	hands:2,
+	saying:"How you doin'?",
+	friends: ["Rex"," Lisa"]
+};
+const woman = {
+	species:"human",
+	name:"Lisa",
+	gender:"female",
+	legs:2,
+	hands:2,
+	saying:"Hello!",
+	friends: ["Joey"," Rex"," Selina"]
+};
+const catWoman = {
+	species:"cat-woman",
+	name:"Selina",
+	gender:"female",
+	legs:2,
+	hands:2,
+	saying:cat.saying,
+	friends: ["Milka"," Lisa"]
+}
+const inhabitants = [dog, cat, man, woman, catWoman];
+const properties = ["species", "name", "gender", "legs", "hands", "saying", "friends"]
```

Comment:
JS allows semicolons ommisiion. Both semicoloned and semicolonless styles are legal. But stick to either consistently.